Documentation/security/network/encryption-ipsec.rst

```diff
@@ -211,15 +211,25 @@ Troubleshooting
 
  * All XFRM errors correspond to a packet drop in the kernel. Except for
    ``XfrmFwdHdrError`` and ``XfrmInError``, all XFRM errors indicate a bug in
-   Cilium or an operational mistake. ``XfrmOutStateSeqError`` and
-   ``XfrmInStateProtoError`` may be caused by operational mistakes, as detailed
-   in the following points.
+   Cilium or an operational mistake. ``XfrmOutStateSeqError``,
+   ``XfrmInStateProtoError``, and ``XfrmInNoStates`` may be caused by
+   operational mistakes, as detailed in the following points.
 
  * If the sequence number reaches its maximum value for any XFRM OUT state, it
    will result in packet drops and XFRM errors of type
    ``XfrmOutStateSeqError``. A key rotation resets all sequence numbers.
    Rotate keys frequently to avoid this issue.
 
+ * After a key rotation, if the old key is cleaned up before the
+   configuration of the new key is installed on all nodes, it will result in
+   ``XfrmInNoStates`` errors. The old key is removed from nodes after 5 minutes
+   by default. By default, all agents watch for key updates and update their
+   configuration within 1 minute after the key is changed, leaving plenty of
+   time before the old key is removed. If you expect the key rotation to take
+   longer for some reason (e.g., in case of clustermesh several clusters need
+   to be updated), you can increase the delay before cleanup with agent flag
+   ``ipsec-key-rotation-duration``.
```

Comment:
Edits for clarity, future tense, no Latin abbreviations. Normally I would review for passive voice, but I think active voice alternatives wouldn't be nearly as clear.
```suggestion
   configuration of the new key is installed on all nodes, it results in
   ``XfrmInNoStates`` errors. The old key is removed from nodes after a default interval
   of 5 minutes. By default, all agents watch for key updates and update their
   configuration within 1 minute after the key is changed, leaving plenty of
   time before the old key is removed. If you expect the key rotation to take
   longer for some reason (for example, in the case of Cluster Mesh if several clusters need
   to be updated), you can increase the delay before cleanup with agent flag
   ``ipsec-key-rotation-duration``.
```

Reply:
> future tense

I really need to be more careful on that. I know it and you've pointed it before :disappointed: 

> no Latin abbreviations

FYI, regarding that and "clustermesh" vs. "Cluster Mesh", the docs are not consistent (even this page isn't).